### ISSUE TITLE
[PM-1033] update monitord to handle updated log messages in dagman.out file

### ISSUE THREAD

**Issue by mayani** [pegasus-isi/pegasus#1147]
Some of the dagman log messages in dagman.out file were updated to have HTCondor instead of Condor. This breaks the monitord parsing regex's and hence it won't be able to parse information from the dagman.out file. 

Email from Kent below:

Karan,

The "Condor" -> "HTCondor" change is from a while ago (8.5.0).  I guess I probably wouldn't have changed it at that point if I knew that you guys were still reading dagman.out (I thought you had switched over to using jobstate.log).

Basically, if there was Condor in a string that was just something informational generated by DAGMan, it's now HTCondor.  But things like paths, binary names, config macros, etc., don't have the "HT".

Kent

**Reporter:** @vahi
**Resolution:** Fixed
**Watchers:**
@rynge
@vahi

**Comment by mayani** [comment]
**Author:** @vahi

Looks like the following messages were updated

01/05/16 08:45:29 Submitting HTCondor Node cleanup_horizontal-clustering-test_0_local job(s)...
01/05/16 08:45:29 Currently monitoring 1 HTCondor log file(s)

and all the ULOG_* event messages such as

01/05/16 08:45:19 Event: ULOG_JOB_TERMINATED for HTCondor Node level2_ID0000015 (1018631.0.0) {01/05/16 08:45:15}
01/05/16 08:45:24 Event: ULOG_POST_SCRIPT_TERMINATED for HTCondor Node level2_ID0000015 (1018631.0.0) {01/05/16 08:45:19}
01/05/16 08:45:29 Event: ULOG_SUBMIT for HTCondor Node cleanup_horizontal-clustering-test_0_local (1018651.0.0) {01/05/16 08:45:29}
01/05/16 08:45:34 Event: ULOG_EXECUTE for HTCondor Node cleanup_horizontal-clustering-test_0_local (1018651.0.0) {01/05/16 08:45:33}

**Comment by mayani** [comment]
**Author:** @vahi

our tests run fine . the regex's were updated in the monitord codebase
